windows/src/buildtools/sentrytool/Keyman.System.AddPdbToPe.pas

```diff
@@ -265,6 +265,9 @@ function AddDebugID(s: TMemoryStream; guid: TGUID; var codeId: string; var isAmd
     s.SetSize(SectionOffset + SectionRawSize);
     FillChar((PByte(s.Memory) + SectionOffset)^, SectionRawSize, 0);
 
+    nh32 := PImageNtHeaders(NativeUInt(nh32)-NativeUInt(dh)+NativeUInt(s.Memory));
+    section := PImageSectionHeader(NativeUInt(section)-NativeUInt(dh)+NativeUInt(s.Memory));
```

Comment:
After `s.SetSize()`, existing pointers are invalidated and need to be recalculated.

In theory I could have also added:
```
    dh := PImageDosHeader(s.Memory);
```

But `dh` is not used after this so it raises a compiler hint.